napari/_qt/qt_main_window.py

```diff
@@ -174,13 +173,22 @@ def _add_file_menu(self):
             lambda: self.qt_viewer._save_layers(selected=False)
         )
 
-        screenshot = QAction('Save Screenshot', self._qt_window)
-        screenshot.setShortcut('Ctrl+Alt+S')
+        screenshot = QAction('Save screenshot...', self._qt_window)
```

Comment:
We should pick a case convention and stick to it... But every app I have checked uses title case in their menubars...

Reply:
Will go back to uppercase and make consistent